index.html

```diff
@@ -11,25 +11,176 @@
+                        <li class="work-detail work-text">2015</li>
+                    </ul>
+                </div>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                <div>
+                    <div class="work-detail-box">
+                        <ul>
+                            <li class="chips">html</li>
+                            <li class="chips">css</li>
+                            <li class="chips">javascript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="work-button"><a class="project-button" href="#">See Project</a></div>
+            </div>
+        </div>
+
+        <div class="about-section">
+            <h1 class="card-title">About <br>Myself</h1>
+            <p>I’m a software developer! I can help you build a product, feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+            <h2>LET’S CONNECT</h2>
+            <ul>
+                <li class="social-list"><img src="images/twitter.png" alt="twitter"></li>
+                <li class="social-list"><img src="images/Linkedin.png" alt="twitter"></li>
+                <li class="social-list"><img src="images/medium.png" alt="medium"></li>
+                <li class="social-list"><img src="images/github.png" alt="github"></li>
+                <li class="social-list"><img src="images/angellist.png" alt="angellist"></li>
+            </ul>
+            <div class="work-button"><a class="project-button" href="#">Get My Resume</a></div>
+            <div class="language-box1">
+                <h3 class="canopy">Languages</h3>
+                <a href="#"><img class="icons" src="images/down-arrow.png" alt="arrow"></a>
+            </div>
+            <ul>
+                <li class="language">
+                    <img src="images/Ellipse1.png" alt="js">
+                    <h3 class="language-text">JavaScript</h3>
+                </li>
+                <li class="language">
+                    <img src="images/Ellipse2.png" alt="js">
+                    <h3 class="language-text">HTML</h3>
+                </li>
+                <li class="language">
+                    <img src="images/Ellipse3.png" alt="js">
+                    <h3 class="language-text">CSS</h3>
+                </li>
+            </ul>
+            <div class="language-box">
+                <h3 class="canopy">Frameworks</h3>
+                <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
+            </div>
+            <div class="language-box">
+                <h3 class="canopy">Skills</h3>
+                <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
+            </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a list, each of them in a li.